docs/dagster-university/pages/dagster-dbt/lesson-6/2-creating-a-simple-incremental-model.md

```diff
@@ -36,7 +36,7 @@ with
             sum(total_amount) / count(*) as average_amount,
             sum(case when duration > 30 then 1 else 0 end) / count(*) as pct_over_30_min
         from trips
-        group by all
+        group by date_of_business
```

Comment:
Oh, I thought the issue was that 

```sql
count(*) - lag(count(*), 1) over (order by date_trunc('day', pickup_datetime)) as trip_count_change,
```

was accidentally persisted into here. If we just delete that column, does the `group by all` still work?